doc/gtc_osiris.rst

```diff
@@ -0,0 +1,28 @@
+=========
+GTC OSIRIS
+=========
+
+
+Overview
+========
+
+This file summarizes several instrument specific
+settings that are related to the GTC/OSIRIS spectrograph.
+
+Common Items
+============
+
+Targets centred on chip 2
++++++++
```

Comment:
rst format requires the `+++` go exactly as long as the line.
Please fix throughout.

Reply:
Fixed (I think).